### PR TITLE
docs: update README.md to reflect esgit

### PR DESCRIPTION
**Pull request by kaf-lamed-beyt**
added esgit &mdash; easy-git

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -284,7 +285,7 @@
 ## <a name="W"> </a>W
 
 - [Waihona](https://github.com/bisohns/waihona) - Rust crates for performing cloud storage actions across AWS, GCP and Azure. **By [@deven96](https://twitter.com/_deven96)**
-- [WatchDog](https://github.com/IzyPro/WatchDog) - WatchDog is a Realtime HTTP (Request & Response) and Exception logger and viewer for ASP.Net Core Web Apps and APIs. **By [@IzyPro_](https://twitter.com/IzyPro_)**
+- [WatchDog](https://github.com/IzyPro/WatchDog) - WatchDog is a Realtime HTTP (Request & Response) and Exception logger and viewer for ASP.Net Core Web Apps and APIs. **By [@IzyPro\_](https://twitter.com/IzyPro_)**
```
Comment (acekyd) [inline README.MD:288]: Is this a typo? the slash in the author's name.

Comment (kaf-lamed-beyt) [reply README.MD:288]: I have no idea. Probably a mistake from me. I can't really remember.

Comment (acekyd) [reply README.MD:288]: Would have to update it so I can merge.

Comment (kaf-lamed-beyt) [reply README.MD:288]: Yep. I'm on it.

**Pull request merged**